Libraries/Serialization/ZeroContainers.hpp

```diff
@@ -143,7 +143,7 @@ struct Policy<Pair<first, second>>
   {
     serializer.Start(Trait<type>::TypeName(), fieldName, Serialization::Trait<type>::Type);
     serializer.SerializeField("key", pair.first);
-    serializer.SerializeField("value", pair.second);
+    serializer.SerializeFieldDefault("value", pair.second, second());
```

Comment:
This at least works for all places used by the engine.
Not defaulting key because it never makes sense to have a keyless map entry.